### DISCUSSION TITLE
Add support for AzureSQL

### DISCUSSION THREAD

**Discussion by miiitch** [directus/directus#6411] [Feature Requests]
Hi,

The MSSQL configuration is not enough for using Azure SQL. The knex config also need an _encrypt_ option flags. I created a commit here for testing: https://github.com/miiitch/directus/commit/88a8b5e80bdf1f9c4fbb986fcd77afcba8d57e87 and I managed to configure the database. 

What do you think?

Regards,

Michel

**Comment by rijkvanzanten** [comment]
Looks good to me!

For what it's worth, any configuration flag to knex can already be passed using `DB_<etc>` here, which means that you can already use it with Azure by using an existing environment configuration + `npx directus bootstrap`, but I agree that it would be nicer to add a dedicated "encrypt" question to the `init` script 🙂

**Reply by miiitch** [reply]
I didn't know this trick. Let me test it and if it works, I would add some lines about it in the documentation.

**Reply by julius-plener-amperias** [reply]
`DB_OPTIONS__ENCRYPT=true` in the `.env` file worked for me. But I get a `Database doesn't have Directus tables installed.` error when installing manually, like described in https://docs.directus.io/guides/installation/manual/

**Reply by julius-plener-amperias** [reply]
Update: I had to run `npx directus bootstrap` to get it running.

**Comment by licitdev** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
